README.md

```diff
@@ -711,6 +712,40 @@ function findInlineScriptsWithNextData() {
 
 console.log(findInlineScriptsWithNextData());
 
+```
+
+### Inline CSS Info and Size
+
+Find all inline style tags and list them in a table with individual and total byte size. Customize the table below.
+
+```javascript
+
+// Wait for the page to fully load
+
+function findAllInlineCSS() {
+  const convertToKb = (bytes) => bytes / 1000;
+  const inlineCSS = document.querySelectorAll("style");
+  let totalByteSize = 0;
+  for (const css of [...inlineCSS]) {
+    const html = css.innerHTML;
+    const size = new Blob([html]).size;
+    css.byteSize = convertToKb(size) + " kb";
```

Comment:
Remove the unit to avoid converting it to a string.
As string, the order doesn't work fine.

![image](https://github.com/nucliweb/webperf-snippets/assets/1307927/cb1996ba-c7a1-45e5-812b-29ad37286381)
> Ordered by "byteSize"

```suggestion
    css.byteSize = convertToKb(size);
```

Reply:
good point, missed that